common/src/main/kotlin/io/homeassistant/companion/android/common/data/prefs/PrefsRepository.kt

```diff
@@ -1,17 +1,47 @@
 package io.homeassistant.companion.android.common.data.prefs
 
+import androidx.appcompat.app.AppCompatDelegate
 import io.homeassistant.companion.android.common.data.integration.ControlsAuthRequiredSetting
 import io.homeassistant.companion.android.common.util.GestureAction
 import io.homeassistant.companion.android.common.util.HAGesture
 
+enum class NightModeTheme(val storageValue: String) {
```

Comment:
I was wondering if this is at the right place if you have suggestions.

Reply:
The enum is OK I think, but having `AppCompatDelegate` in here is a bit weird. Why not put that in NightModeManager (in the app module)?

Reply:
OK let's just continue that conversation in https://github.com/home-assistant/android/pull/5609#discussion_r2244737531, two review comment threads about the same subject.